Sprint-2/debug/0.js

```diff
@@ -1,7 +1,12 @@
 // Predict and explain first...
 
 function multiply(a, b) {
-  console.log(a * b);
+  return (a * b);
 }
 
 console.log(`The result of multiplying 10 and 32 is ${multiply(10, 32)}`);
+
+
+// Here we defined function to just print the result not return it, therefore it returns undefined in Javascript by default.
```

Comment:
Oh, I'm a bit lost here. Let's clarify this piece by piece.
```suggestion
// The function "multiply" does not return anything; therefore it returns undefined.
```

Reply:
Thank you. clear now.